crates/context_aware_config/src/api/context/handlers.rs

```diff
@@ -75,39 +73,37 @@ pub fn endpoints() -> Scope {
         .service(list_contexts)
         .service(get_context_from_condition)
         .service(get_context)
-        .service(priority_recompute)
         .service(weight_recompute)
 }
 
 type DBConnection = PooledConnection<ConnectionManager<PgConnection>>;
 
-pub fn validate_dimensions_and_calculate_priority(
+pub fn validate_dimensions(
     object_key: &str,
     cond: &Value,
     dimension_schema_map: &HashMap<String, DimensionData>,
-) -> superposition::Result<i32> {
-    let get_priority = |key: &String, val: &Value| -> superposition::Result<i32> {
+) -> superposition::Result<()> {
+    let check_dimension = |key: &String, val: &Value| -> superposition::Result<()> {
         if key == "var" {
             let dimension_name = val
                 .as_str()
                 .ok_or(bad_argument!("Dimension name should be of `String` type"))?;
             dimension_schema_map
                 .get(dimension_name)
-                .map(|dimension_val| &dimension_val.priority)
+                .map(|_| Ok(()))
```

Comment:
nab (not a blocker): we can remove this?

Reply:
since in the else case it is a recursive call which has result<()> type